src/test/java/org/jabref/logic/l10n/LocalizationParser.java

```diff
@@ -223,6 +226,17 @@ public boolean containsKey(String key) {
                      .collect(Collectors.toList());
     }
 
+    private static void setStaticLoad(FXMLLoader loader) {
+        // Somebody decided to make "setStaticLoad" package-private, so let's use reflection
+        try {
+            Method method = FXMLLoader.class.getDeclaredMethod("setStaticLoad", boolean.class);
```

Comment:
Maybe we can add our arguments to the PR discussion? https://github.com/javafxports/openjdk-jfx/pull/44/files